deep_qa/layers/encoders/positional_encoder.py

```diff
@@ -57,13 +57,8 @@ def my_keras_cumsum(tensor, axis=0):
             """
             # Putting the imports here so that tests and things don't have to run them, unless
             # they're using this encoder with the particular backend.
-            if K.backend() == "tensorflow":
-                import tensorflow as tf
-                return tf.cumsum(tensor, axis=axis)
-            else:
-                import theano.tensor.extra_ops as T
-                return T.cumsum(tensor, axis=axis)
-
+            import tensorflow as tf
```

Comment:
We can just remove this whole `my_keras_cumsum` method, because there is now a `K.cumsum`, which we can use.